package com.revature.utils.exceptions;

public class DataSourceException extends RuntimeException {

    public DataSourceException(String msg, Throwable cause) {
        super(msg, cause);
    }

}
